### PR TITLE
Feature/add tooltip

### PR DESCRIPTION
**Pull request by miladkdz**

### CODE REVIEW

File: demo/incubator-avatar-basic-demos.html [hunk 1]
```diff
@@ -13,7 +13,7 @@ <h3>Basic usage</h3>
         </incubator-avatar>
         <incubator-avatar name="William Davids">
         </incubator-avatar>
-        <incubator-avatar name="Bruce Williams" abbr="BJ">
+        <incubator-avatar no-tooltip name="Bruce Williams" abbr="BJ" tooltip-position="top" tooltip-align="left">
```
Comment (peancored) [inline demo/incubator-avatar-basic-demos.html]: What's the point of using tooltip-position and tooltip-align with no-tooltip? Can we make separated demos for positioning?

File: src/incubator-avatar.html [hunk 2]
```diff
@@ -126,6 +164,10 @@
 
           this.abbr = name.split(' ').map((word) => word[0]).join('');
         }
+
+        _displayTooltip() {
```
Comment (peancored) [inline src/incubator-avatar.html]: The method's name looks like the method opens the tooltip or something. Consider renaming it, please.

File: src/incubator-avatar.html [hunk 3]
```diff
@@ -116,7 +152,9 @@
         }
 
         _setTitle(name) {
-          this.title = name;
+          if(!this._displayTooltip()) {
```
Comment (peancored) [inline src/incubator-avatar.html]: I think you should check the `this.noTooltip` variable here instead

File: demo/incubator-avatar-basic-demos.html (hunk 1 above)
Comment (miladkdz) [reply demo/incubator-avatar-basic-demos.html]: I was planning to change this myself.

File: src/incubator-avatar.html (hunk 3 above)
Comment (miladkdz) [reply src/incubator-avatar.html]: The title attribute should not be set if we are going to show the tooltip.

**Pull request merged**